bibformat/format_elements/bfe_INSPIRE_enhanced_marcxml.py

```diff
@@ -121,6 +122,14 @@ def get_matched_id(subfields):
     return None
 
 
+def salvage_deleted_record_from_history(recid):
+    return create_record(decompress(run_sql("SELECT marcxml FROM hstRECORD WHERE id_bibrec=%s ORDER BY job_date DESC LIMIT 1", (recid, ))[0][0]))[0]
```

Comment:
Mmh. I took as example record 2 and found that it was stored deleted in `bibfmt`, while expaned in `hstRECORD`, so I concluded `bibfmt` is storing the already purged version. Maybe we have changed this at some point, but seem safer to rely on `hstRECORD` which always have the non-purged version.